pkgs/development/python-modules/imgtool/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, cbor2
+, click
+, cryptography
+, intelhex
+}:
+
+buildPythonPackage rec {
+  pname = "imgtool";
+  version = "1.10.0";
+  format = "setuptools";
```

Comment:
```suggestion
  pyproject = true;
```
We have a new parameter for the format.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

Reply:
Done, `setuptools` added in `nativeDependencies`.